components/brave_rewards/resources/extension/brave_rewards/_locales/en_GB/messages.json

```diff
@@ -168,10 +168,10 @@
     "description": ""
   },
   "braveContributeTitle": {
-    "message": "Auto-Contribute",
+    "message": "Contribute",
     "description": ""
   },
-  "contributeNotification": {
+  "contributeNotificationSuccess": {
```

Comment:
These shouldn't be changed in en_GB they will just be overwritten on the next transifex update.

Reply:
no worries, this will be fixed in https://github.com/brave/brave-core/pull/896